From 48b4e1a208fa3c3ae6bcf5b9583b8bf027a00a8e Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 11:14:10 +0700
Subject: [PATCH 276/475] ARM: dts: r8a7743: Add fdp{0,1} nodes

Add Fine Display Processor (FDP module) nodes for the
Renesas FDPM kernel module driver probe.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index e040b97..4efcd1e 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -1680,6 +1680,22 @@
 			status = "disabled";
 		};
 
+		fdp0: fdpm@fe940000 {
+			compatible = "renesas,fdp1";
+			reg = <0 0xfe940000 0 0x2400>;
+			interrupts = <0 262 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7743_CLK_FDP1_0>;
+			power-domains = <&cpg_clocks>;
+		};
+
+		fdp1: fdpm@fe944000 {
+			compatible = "renesas,fdp1";
+			reg = <0 0xfe944000 0 0x2400>;
+			interrupts = <0 263 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7743_CLK_FDP1_1>;
+			power-domains = <&cpg_clocks>;
+		};
+
 		du: display@feb00000 {
 			compatible = "renesas,du-r8a7743";
 			reg = <0 0xfeb00000 0 0x40000>,
-- 
2.7.4

